### ISSUE TITLE
Frontend people needed for bkkhack collaboration app

### ISSUE THREAD

**Issue by djay** [bkkhack/hacknights#163]
Simple way for people to post
- what they are working on
- where they sit
- how to get in touch

How it could work:

- login via github auth
- view a background image which is a seating map
- drag your avatar where you want to sit
- add some text on what you want to work on. Collaboration indicated via proximity and text only.
- clicking on someone else initiates gitter chat or github profile
- bonus points for someway to indicate online status
- admin users can move anyone + upload/move the background

Non functional requirements
- All data stored in github so no server required. Can be hosted on github pages
   - Map can be stored in an issue description. + link to github page with this app.
   - Each comment stores the position and tags of the a single user. Can subscribe to changes to comments via https://developer.github.com/v3/activity/events/types/#issuecommentevent
- Frontend in something good but hopefully works well on mobile as well.

**Comment by djay** [comment]
going work on this @ches ?

**Comment by ches** [comment]
I might. It may not get much further than me fooling around with a new web framework, and fancy frontend isn't my forte, so if anyone wants to try prototyping a seat map UI be my guest and let's chat.

**Comment by aljones15** [comment]
Maybe Will could work on the seat ui in his google Polymer project?

**Comment by djay** [comment]
put in some basic functional requirements of how it can work.

**Comment by JeremyRms** [comment]
Still need help?

**Comment by djay** [comment]
yes @JeremyZr . Trying to learn a frontend framework or already know one? Come and find me tonight.

**Comment by JeremyRms** [comment]
@djay I know Angular 1 and VueJs

**Comment by natsuhikok** [comment]
Im working as web designer/coder(html/css/jquery) and learning react.js recently. 
also Im interesting in other frontend frameworks.
Can I join today?

**Comment by djay** [comment]
Thanks @JeremyZr @natsuhikok . Sounds like we have a team! with react, angular and vue. Let's hope it doesn't take all night to pick one. see you guys there.

**Comment by waf** [comment]
I'll be hacking on this again!

**Comment by djay** [comment]
@waf let know when you have got this up and running on our github so I can test it out. Also let me know if you want me to setup the heroku gatekeeper instance on my account.
